tests/test_deku_id.rs

```diff
@@ -100,3 +100,15 @@ fn test_litbytestr() {
 
     assert_eq!(b"123", TestEnumArray::VarA.deku_id().unwrap().as_ref());
 }
+
+#[test]
+#[should_panic(expected = "called `Result::unwrap()` on an `Err` value: IdVariantNotFound")]
```

Comment:
Good test, I'm sure this could be supported with deku_id() but I'm not sure it's really needed as the discriminant is already available? If anything, it would be done for consistency... Could be done in a separate PR.

Reply:
That was my thoughts exactly on that issue.